server/schema/questions/index.js

```diff
@@ -0,0 +1,7 @@
+const createQuestionsSchema = require('./createQuestions')
```

Comment:
this is only for adding a single question right??
then it should be `createQuestionSchema` instead of `createQuestionsSchema`

the file name is also to be changed in this case

Reply:
yep My bad !